README.md

```diff
@@ -2,7 +2,7 @@ Software Engineering
 
 ## CSCI 430 (Spring 2025):
 
-* [PassGuard](teams/passguard.md)
+* [Snapshot](teams/Snapshot.md)
```

Comment:
The current commit removes an existing team. You should add your team without removing PassGuard.